src/Classes/ServiceAPI/MyRadio_Track.php

```diff
@@ -619,27 +619,36 @@ public static function cacheAndIdentifyUploadedTrack($tmp_path)
 
         $getID3 = new \getID3();
         $fileInfo = $getID3->analyze(Config::$audio_upload_tmp_dir.'/'.$filename);
+        $getID3_lib = new \getID3_lib();
+        $getID3_lib->CopyTagsToComments($fileInfo);
```

Comment:
Is it necessary to assign this to a variable? IIRC you can do something like `(new \getID3_lib())->...` or maybe there's something even better - is it static, perhaps?